File: 1672-richest-customer-wealth/1672-richest-customer-wealth.java
class Solution {
    public int maximumWealth(int[][] accounts) {
        int max = 0;
        int[] subAccount = new int[accounts[0].length];

        for(int[] customer : accounts) {
            int currentCustomerWealth = 0;
            for(int bank : customer) {
                currentCustomerWealth += bank;
            }
            max = Math.max(max, currentCustomerWealth);
        }

        return max;
    }
}

File: 1672-richest-customer-wealth/1672-richest-customer-wealth.js
/**
 * @param {number[][]} accounts
 * @return {number}
 */
var maximumWealth = function(accounts) {
    let max=-1
    let temp=0
    accounts.forEach((v)=>{
        
        temp=v.reduce((a,b)=>a+b)
        if(temp>max) max=temp
    })
    
    return max
};